packages/@react-spectrum/numberfield/test/NumberField.test.js

```diff
@@ -1100,13 +1134,16 @@ describe('NumberField', function () {
     ${'positive lower positive upper'} | ${'up'}      | ${{minValue: 5, maxValue: 15}}                | ${5}
     ${'positive lower positive upper'} | ${'down'}    | ${{minValue: 5, maxValue: 15}}                | ${15}
   `('$direction $Name starts from the right place', ({direction, props, expected}) => {
-    let {textField} = renderNumberField({onChange: onChangeSpy, ...props});
+    let {textField} = renderNumberField({onChange: onChangeSpy, onBlur: onBlurSpy, onKeyUp: onKeyUpSpy, ...props});
     let key = direction === 'up' ? 'ArrowUp' : 'ArrowDown';
     act(() => {textField.focus();});
     fireEvent.keyDown(textField, {key});
     fireEvent.keyUp(textField, {key});
     expect(onChangeSpy).toHaveBeenCalledWith(expected);
+    expect(onBlurSpy).not.toHaveBeenCalled();
     act(() => {textField.blur();});
+    expect(onBlurSpy).toHaveBeenCalledTimes(1);
+    expect(onKeyUpSpy).toHaveBeenCalledTimes(1);
```

Comment:
why do so many tests check keyupspy?

Reply:
I wanted to check other events that were previously missing. I'll trim the number of these down.